scripts/core/superdesk-api.d.ts

```diff
@@ -879,6 +879,10 @@ declare module 'superdesk-api' {
         };
         confirm_spike: boolean;
         language: string; // default client language
+        editor3: {
+            /** set if browser spellcheck is enabled */
+            spellCheck: boolean;
```

Comment:
Can you rename it to `enableBrowserSpellchecker`? We have 2 spellcheckers so it'd be clearer which one is being referred to. I see you clarified it in the comment. Why not put it in the config name?